hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieCompactionConfig.java

```diff
@@ -188,11 +188,11 @@ public class HoodieCompactionConfig extends HoodieConfig {
       .withDocumentation("Log compaction can be scheduled if the no. of log blocks crosses this threshold value. "
           + "This is effective only when log compaction is enabled via " + INLINE_LOG_COMPACT.key());
 
-  public static final ConfigProperty<String> USE_LOG_RECORD_READER_SCAN_V2 = ConfigProperty
-      .key("hoodie.log.record.reader.use.scanV2")
+  public static final ConfigProperty<String> ENABLE_OPTIMIZED_LOG_BLOCKS_SCAN = ConfigProperty
+      .key("hoodie.optimized.log.blocks.scan.enable")
```

Comment:
how about `hoodie.optimized.log.block.scan.enabled`

Reply:
we should make all the enabling configs align and use `*.enabled`

Reply:
in all of metadata configs, we end with "enable". 
hoodie.metadata.enable
hoodie.metadata.metrics.enable
hoodie.metadata.index.bloom.filter.enable
hoodie.metadata.index.column.stats.enable

and this new config is mainly for metadata table and so keep both write config and metadata config in sync, went w/ that

Reply:
lets leave it as is.